spew/dump.go

```diff
@@ -507,3 +511,26 @@ get the formatted result as a string.
 func Dump(a ...interface{}) {
 	fdump(&Config, os.Stdout, a...)
 }
+
+// isZero is a helper function that checks whether a reflect.Value has the zero value for its type.
+func isZero(v reflect.Value) bool {
+	switch v.Kind() {
+	case reflect.Array, reflect.Map, reflect.Slice, reflect.String:
```

Comment:
```suggestion
	case reflect.String:
```
and never ignore empty arrays as explained above.